changelog

```diff
@@ -52,7 +52,8 @@ Version 1.13.10+dev:
    * Greatly improved behavior of sliders.
    * Fixed crash when modifying an existing friend entry in Preferences.
    * Fixed players being unable to start campaigns in MP mode.
- * WML Engine:
+ * WML Engine:  
+ * added a major_amla option in advancement for what XP bar remain teal/white with AMLA (Quenoth Youth in Utbs by exemple and similary unit line)
```

Comment:
Should be "for example" and "similar unit line" in the parenthetical. Also, "for what XP bar remain ..." doesn't make much sense, though I think I can decipher the meaning.